### PR TITLE
UrlScan Error fixes

### PR DESCRIPTION
**Pull request by paulusaltus**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
n/a

## Description
Specific error messages returned by the API were not being handled properly, causing the function to fail b/c it was expected to return 3 values, and was only returning 2.

## Minimum version of Cortex XSOAR
N/A-fix only

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by paulusaltus** [comment]
RE LGTM error:
Pep8 specifies "The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces."
Let me know any questions or concerns, or if there's a different way to do it via convention (using the slash '\' at end of line gave linter errors so I had removed it),

**Comment by ostolero** [comment]
Hi Paul, thank you for this contribution!
I see this pack is a partner's pack, meaning UrlScan is responsible for this code and any changes made to it.
I'll try and contact them to see if this contribution can be received and will update you as soon as I have an answer.

**Comment by heipei** [comment]
Thanks @paulusaltus we had these on our radar ourselves, really appreciate you stepping up. Fine to have these merged!

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.py [hunk 1]
```diff
@@ -21,8 +21,11 @@
 urllib3.disable_warnings()
 
 '''GLOBAL VARS'''
-BLACKLISTED_URL_ERROR_MESSAGE = 'The submitted domain is on our blacklist. ' \
-                                'For your own safety we did not perform this scan...'
+BLACKLISTED_URL_ERROR_MESSAGES = [
+    'The submitted domain is on our blacklist. '
+    'For your own safety we did not perform this scan...',
```
Comment (ostolero) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.py:26]:
```suggestion
    'The submitted domain is on our blacklist. For your own safety we did not perform this scan...',
```

I think this syntax should pass validation and is clearer

**Review by ostolero — COMMENTED** [review]
Hi @paulusaltus these look great, I just added a suggestion regarding the string error, let me know if this works as expected :) 
I think we can proceed to a demo, I'll contact you via slack to arrange it.

**Comment by paulusaltus** [comment]
sounds great, lets go ahead and apply that suggestion to prefer the single line

**Comment by ostolero** [comment]
Great! all we have to do now is schedule our demo :)

**Review by ostolero — APPROVED** [review]
Reviewed a video demo and everything's looks great 👍

**Pull request merged**